test/formatting.test.ts

```diff
@@ -2,8 +2,37 @@
 import * as util from '../src';
 import { EMPTY_FIELD } from '../src';
 
+const EMPTY_VALUES = [
+  '             ',
+  ' ',
+  '',
```

Comment:
is there a reason for the different # of spaces in the empty string?

Reply:
Just seemed like a fun extra thing to test for. A very long but technically empty string. Not too dissimilar from ' ' but I have some code that strips the string down no matter the length so I wanted to test it.

> A QA Engineer walks into a bar. The QA engineer orders a beer. Orders 0 beers. Orders 999999999 beers. Orders a lizard. Orders -1 beers. Orders a sfdeljknesv.

Reply:
gotcha!!!!